contrib/julia-config.jl

```diff
@@ -36,7 +37,7 @@ end
 function ldflags()
     fl = "-L$(shell_escape(libDir()))"
     if Sys.iswindows()
-        fl = fl * " -Wl,--stack,8388608"
+        fl = fl * " -Wl,--export-all-symbols -Wl,--stack,8388608"
```

Comment:
Can you open an issue for this and assign me? I know we used to emit a correct file, so it seems like that has regressed.